package.json

```diff
@@ -20,11 +20,11 @@
     "remove-dist": "rimraf ./dist",
     "prebuild": "npm run clean-dist && npm run lint && npm run test",
     "build": "babel-node tools/build.js && npm run open:dist",
-    "test": "mocha tools/testSetup.js src/**/*.spec.js --reporter progress",
-    "test:cover": "babel-node node_modules/isparta/bin/isparta cover --root src --report html mocha -- --require ./tools/testSetup.js src/**/*.spec.js --reporter progress",
-    "test:cover:travis": "babel-node node_modules/isparta/bin/isparta cover --root src --report lcovonly _mocha -- --require ./tools/testSetup.js src/**/*.spec.js && cat ./coverage/lcov.info | node_modules/coveralls/bin/coveralls.js",
+    "test": "node --harmony_proxies node_modules/jest/bin/jest",
+    "test:cover": "npm run test -- --coverage ",
+    "test:cover:travis": "npm run test:cover && cat ./coverage/lcov.info | node_modules/coveralls/bin/coveralls.js",
```

Comment:
I removed that and the lcov.info is still being produced. Will confirm the coverage once checks are completed.